test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
It's a bit confusing, I actually reverted your commit 3295269ebd372728948845121147b8a66e4c76f5 and ran the adapter test suite against both rgeo `0.4.0` and `0.6.0`. In both cases tests passed.